test/e2e/utils_test.go

```diff
@@ -399,3 +401,27 @@ func verifyCertificateRenewed(ctx context.Context, secretName, namespace string,
 		return true, nil
 	})
 }
+
+// create randomized string
+func randomStr(size int) string {
+	char := "abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789"
+	rand.NewSource(time.Now().UnixNano())
+	var s bytes.Buffer
+	for i := 0; i < size; i++ {
+		s.WriteByte(char[rand.Int63()%int64(len(char))])
+	}
+	return s.String()
+}
+
+//replace string in bytes of ReadFile
+func ReplaceStrInFile(oldStr []string, newStr []string, fileName string) ([]byte, error) {
```

Comment:
This file's all functions are initial lower case. I do not see the necessity why this function needs initial upper case.
```suggestion
func replaceStrInFile(oldStr []string, newStr []string, fileName string) ([]byte, error) {
```

Reply:
updated.